lametro/templates/lametro/legislation.html

```diff
@@ -169,6 +168,22 @@ <h3 class="no-pad-bottom"><i class="fa fa-file-text" aria-hidden="true"></i> Rel
                 </div>
                 <div class="divider"></div>
             {% endif %}
+
+            {% if legislation.topics %}
+            <br>
```

Comment:
i moved the `<br>` to inside the conditionals to prevent two of them stacking up.